File: Programmers/Lv.0/문자열 묶기/181855.js
function solution(strArr) {
    let answer = 0;
    
    const lengthArr = new Array(30).fill(0);
    strArr.forEach(v => {
        lengthArr[v.length - 1]++;
    });
    
    answer = Math.max(...lengthArr);
    
    return answer;
}

File: Programmers/Lv.0/문자열 묶기/181855.java
import java.util.Arrays;

class Solution {
    public int solution(String[] strArr) {
        int answer = 0;
        
        int[] lengthArr = new int[30];
        for (String str : strArr) {
            lengthArr[str.length() - 1]++;
        }
        
        Arrays.sort(lengthArr);
        
        answer = lengthArr[29];
        
        return answer;
    }
}
